2-ui/99-ui-misc/02-selection-range/article.md

```diff
@@ -495,36 +495,36 @@ The exception is some selection methods, that replace the existing selection, su
 
 ## Selection in form controls
 
-Form elements, such as `input` and `textarea` provide [special API for selection](https://html.spec.whatwg.org/#textFieldSelection), without `Selection` or `Range` objects. As an input value is a pure text, not HTML, there's no need for such objects, everything's much simpler.
-
-Properties:
-- `input.selectionStart` -- position of selection start (writeable),
-- `input.selectionEnd` -- position of selection end (writeable),
-- `input.selectionDirection` -- selection direction, one of: "forward", "backward" or "none" (if e.g. selected with a double mouse click),
+عناصر فرم، مانند `input` و textarea [special API for selection](https://html.spec.whatwg.org/#textFieldSelection) را بدون اشیاء `Selection` یا `Range` ارائه می‌کنند. از آنجایی که مقدار ورودی یک متن خالص است، نه HTML، نیازی به چنین اشیایی نیست، همه چیز بسیار ساده تر است.
 
-Events:
-- `input.onselect` -- triggers when something is selected.
-
```

Comment:
Please check these lines as well. and all the lines. the line numbers on both sides for every single line should be the same like this:

<img width="162" alt="image" src="https://github.com/javascript-tutorial/fa.javascript.info/assets/20593549/088b8965-e760-4025-b6b6-277363680dc8">

Reply:
I think it is correct, for example line 498 and 499 are both the same and also other lines.
@mahdyar